effi-ui-html-editor.js

```
@@ -0,0 +1,1281 @@
+                    newElement = '<hr style="margin: 20px 0;">';
+                    break;
+                case 'container':
+                    newElement = `<div class="container" style="padding: 20px; background: #f5f5f5;">
+                        <div class="drop-zone" data-zone="container">גרור רכיבים לכאן</div>
+                    </div>`;
+                    break;
+                case 'columns':
+                    newElement = `<div class="columns" style="display: flex; gap: 20px;">
+                        <div class="column" style="flex: 1;">
+                            <div class="drop-zone" data-zone="col1">עמודה 1</div>
+                        </div>
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)